app/src/test/java/com/androidavanzado/covid19stadistics/util/DateTomorrow.kt

```diff
@@ -0,0 +1,28 @@
+package com.androidavanzado.covid19stadistics.util
+
+import android.annotation.SuppressLint
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+
+@SuppressLint("SimpleDateFormat")
+class DateTomorrow {
```

Comment:
This class I think it's going to be used only in StadisticsViewModelTest so why not changing it's name to StadisticsViewModelTestDateUtils ?

Reply:
done, thanks